components/Footer.vue

```diff
@@ -17,6 +17,7 @@
           </span>
           <span class="c-footer__side-lang">
             <lang-picker></lang-picker>
+            <a href="https://crowdin.com/profile/coderdojo" class="c-footer__side-lang-translate">{{ $t('Help us with translations') }}</a>
```

Comment:
I feel like this should line up with the bottom of the about coderdojo paragraph. otherwise it looks to close to the need help links

Reply:
The position is up to discussion in the Trello already ;)